Taskfile.yml

```diff
@@ -199,7 +199,7 @@ tasks:
 
   evg-test-deployed-lambda-aws: bash ${DRIVERS_TOOLS}/.evergreen/aws_lambda/run-deployed-lambda-aws-tests.sh
 
-  evg-gather-test-suites: find . -name \*.suite | xargs tar czf test_suite.tgz
+  evg-gather-test-suites: find . -name \*.suite | xargs tar czf test_suite.tgz || true
```

Comment:
Why would there not be *.suite files? Aslo, the current solution doesn't fail when I run it locally. Does it fail in a spawn host?

```
❯ task evg-gather-test-suites
task: [evg-gather-test-suites] find . -name \*.suite | xargs tar czf test_suite.tgz
```

If this is a legitimate use case, then suggest we prevent the tar command from running when no files are found rather than appending || true to mask errors. Can use ` --no-run-if-empty` 

```bash
find . -name \*.suite  | xargs --no-run-if-empty tar czf test_suite.tgz
```